exercises/practice/pop-count/.meta/src/example.cr

```diff
@@ -0,0 +1,10 @@
+class PopCount
+  def self.egg_count(number : Int32) : Int32
+    count : Int32 = 0
```

Comment:
```suggestion
    count = 0_i32
```

I'm fine either way, but this is an alternative that you may prefer

Reply:
Crystal will assign by default int32 so I think it is also fine to just write it as `count = 0`.